kong.conf.default

```diff
@@ -234,7 +234,19 @@
                          # capabilities) pulling its configuration changes
                          # from the database.
                          #
-                         # Example: `stream_listen = 127.0.0.1:8444 http2 ssl`
+                         # Example: `admin_listen = 127.0.0.1:8444 http2 ssl`
+
+ #status_listen = off    # Comma-separated list of addresses and ports on
+                         # which the Status interface should listen.
+                         # Status interface allows you to
```

Comment:
typo: The* Status interface. Also not a fan of addressing directly to the reader ("you"), but the pattern is already established in a few other configuration properties...